src/Holmakefile.inc

```diff
@@ -18,8 +20,8 @@ default: all
 # ----------------------------------
 ifdef POLY
 
-$(NEWHOLHEAP): $(TARGETS) $(HOLHEAP)
```

Comment:
don't we want that the new holheap is rebuilt in case thereferenced holheap changes for some reason?

Reply:
Indeed, you're right. I removed that because Holmake said "Don't know how to make necessary target". We should add it back.

Reply:
```suggestion
$(NEWHOLHEAP): $(TARGETS) $(HOLHEAP)
```
(trying GitHub feature)

Reply:
interesting that it worked before then!